discussion/validators.py

```diff
@@ -0,0 +1,19 @@
+import os
+from django.core.exceptions import ValidationError
+def validate_file_extension(file):
+    """
+    Validates the file extension and type
+    """
+
+    valid_extensions = ['.pdf', '.png', '.jpg']
+    valid_content_types = ['application/pdf', 'image/png', 'image/jpg']
+
+    ext = os.path.splitext(file.name)[1]  # [0] returns path+filename
+    if ext.lower() not in valid_extensions:
+        raise ValidationError(f'{ext.lower()} is not a supported file extension.')
+
+    content_type = file.content_type
```

Comment:
Unfortunately it is possible to change the content-type of a request and bypass this check:
See https://owasp.org/www-community/vulnerabilities/Unrestricted_File_Upload

- [ ] Need another method for validating file type